automation/main.py

```diff
@@ -325,7 +325,7 @@ def process_sdk(operation: OperationConfiguration, sdk: SdkConfiguration):
                     published_at = datetime.fromisoformat(release['published_at'].replace('Z', '+00:00'))
                     if operation.date_start < published_at < operation.date_end:
                         release_tag = release['tag_name']
-                        if re.match(sdk.release_tag.regex_match, release_tag):
+                        if re.match(sdk.release_tag.regex_match, release_tag) and release_tag == 'sdk/resourcemanager/containerservice/armcontainerservice/v0.2.1':
```

Comment:
for test

in future will at least add a configure setting for this kind of testing